test_support/src/lib.rs

```diff
@@ -82,50 +82,36 @@ pub fn retry<T, E>(
     attempts: u32,
     retry_delay_in_secs: u64,
```

Comment:
[Optional] This function could be more general purpose if it took a Duration instead of an integer. Then you wouldn't have to worry about mixing up attempt numbers and sleep time as the caller would have `Duration::from_secs(1)` (or whatever) in it directly.

The constant naming `DEFAULT_RETRY_DELAY_IN_SECONDS` was a bit of a hint that this could maybe be represented as a more specific type.